docs/source/background/scf_notes.md

```diff
@@ -0,0 +1,86 @@
+<!--
+  ~ Copyright 2025 NWChemEx-Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+-->
+
+# SCF Procedure
+
+## SCF Driver
+
+1. Obtain the AO parameters as well as the chemical system (inputs)
+2. Assign the AO parameters as atomic orbitals
+3. Create the Hamitonian from the Chemical System (Submod "Hamiltonian" used, currently only the BO Approximation from NUX)
+    1. For the BO Approximation Hamiltonian:
+        1. From the Chemical System, the Electrons and Nuclei are used to form the Hamiltonian
+        2. Each electron gets a 1 electron kinetic energy term
+        3. Each nuclei gets an Electron-Nuclei potential energy term
```

Comment:
Each electron-nuclei pair gets a term. Nuclei don't get a term unless there are electrons present (or if the code tries to give the nuclei a term when no electrons are present it shouldn't).